src/main/java/uk/gov/companieshouse/api/accounts/AccountsApplication.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.companieshouse.api.accounts;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+@SpringBootApplication
+public class AccountsApplication extends WebMvcConfigurerAdapter {
+
+    public static void main(String[] args) {
+
+        // Returns the configured ${PORT} value in the `start.sh` script - passed in under `server.port`.
+        // If no port is configured, return null
```

Comment:
This should not reference the start script. If we change our deployment strategy then this comment would be incorrect.

Reply:
done